selfservice/strategy/oidc/strategy.go

```diff
@@ -457,6 +457,16 @@ func (s *Strategy) handleError(w http.ResponseWriter, r *http.Request, f flow.Fl
 		// Reset all nodes to not confuse users.
 		// This is kinda hacky and will probably need to be updated at some point.
 
+		if errors.Is(err, registration.ErrDuplicateCredentials) {
+			rf.UI.Messages.Add(text.NewErrorValidationDuplicateCredentials())
+			lf, err := s.registrationToLogin(w, r, rf, provider)
+			if err != nil {
+				return err
```

Comment:
not sure what to do here with the `err`, I guess it is okay to return as is?

Reply:
When would this error happen? Only if creating the login flow somehow failed, right?

Reply:
Yeah, also this gets confusing since we call the `handleError` function inside this function.

Reply:
>Yeah, also this gets confusing since we call the handleError function inside this function.

That's what I meant with my other comment. Maybe it's possible to not call `handleError` in the function?

Reply:
```suggestion
				return s.handleError(err ! ...)
```

I think you need to handle the error here, now too.

Reply:
this is inside of the `handleError` function :)

Should probably not call it again recursively

Reply:
lol
I am blind. Sorry.

Reply:
it's all good, I was also tempted at wrapping it at one point before realising it is inside of `handleError` :laughing: